pkg/initialisation/init_data.go

```diff
@@ -131,6 +132,21 @@ func (i *InitData) Init(ctx context.Context, invoker constants.Invoker) {
 	}
 	i.Result.AddWarnings(errorsAndWarnings.Warnings...)
 	i.Client = client
+	// if the cache is set on the workspace profile
+	// and is set to false, then override the default cache setting of the connection
+	if viper.IsSet(constants.ArgClientCacheEnabled) && !viper.GetBool(constants.ArgClientCacheEnabled) {
+		if e := i.Client.CacheOff(ctx); e != nil {
```

Comment:
e???

Reply:
I thought we were going to remove the cache methods from the client and make them funcitons accepting connections

That was probably on Puskars branch - but we need it here too